Dev/GitHub/github-review-requests-TS-5m.ts

```diff
@@ -51,7 +54,7 @@ class DataFetcher {
 
   async fetch({ token, username, filters }: Init): Promise<SearchResults> {
     const { data } = await axios.post(
-      `https://api.github.com/graphql`,
+      this.apiEndpoint,
```

Comment:
Alternatively, if you don't want the main GitHub URL in the xbar variable declaration, you should probably have the default here:

```suggestion
      this.apiEndpoint ?? 'https://api.github.com/graphql',
```